articles/migrations/index.md

```diff
@@ -21,25 +21,14 @@ During the grace period, customers are informed via dashboard notifications and
 
 If you need help with the migration, create a ticket in our [Support Center](${env.DOMAIN_URL_SUPPORT}).
 
+---
```

Comment:
I'd remove the horizontal lines. This is not something we normally use, and if you think we should, we'd better follow up with design before start using them

Reply:
I can't imagine what the problem with using an `hr` is, or why it would require design's input... they're perfectly normal demarcation characters... but I will remove them.